presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -366,6 +365,21 @@ void PrestoServer::run() {
             .sendWithEOM();
       });
 
+  if (systemConfig->enableRuntimeMetricsCollection()) {
+    enableWorkerStatsReporting();
+    if (folly::Singleton<velox::BaseStatsReporter>::try_get()) {
+      httpServer_->registerGet(
+          "/v1/info/metrics/",
```

Comment:
Is it intentional there is a `/` at the end? It makes a difference in the request. The following won't work
```
curl -v http://127.0.0.1:7777/v1/info/metrics
```
and results in `Unsupported HTTP request`.
It has to be
```
curl -v http://127.0.0.1:7777/v1/info/metrics/
```